devops/values/staging.yaml

```diff
@@ -266,6 +266,8 @@ replybot:
     value: "http://gbv-botserver"
   - name: FACEBOOK_GRAPH_URL
     value: "https://graph.facebook.com/v8.0"
+  - name: FORMCENTRAL_URL
```

Comment:
Looks good, but will need to change the values in `test.yaml` in order to get the e2e tests to pass!

Reply:
`FORMCENTRAL_URL ` is already in `test.yaml`, https://github.com/vlab-research/fly/blob/main/devops/values/test.yaml#L139-L140.